lib/request_handler/fieldsets_parser.rb

```diff
@@ -29,13 +29,18 @@ def run
     private
 
     def parse_options(type, values)
+      raise ExternalArgumentError, fieldsets: "invalid type: <#{type}>" if allowed[type] == false
```

Comment:
So only allow true and enum as validation Classes? and in all other cases raise_invalid_field_option?